builders/build/eth-api/dev-env/foundry.md

```diff
@@ -387,7 +425,7 @@ abi.encode(100, true, "Develop on Moonbeam")
 2. 加载存储的状态
 
     ```bash
-    chisel load
+    chisel load 1
```

Comment:
on line 437, the image should be removed and replaced with:

--8<-- 'code/builders/build/eth-api/dev-env/foundry/terminal/save-state.md'